SPECS/initramfs/initramfs.spec

```diff
@@ -101,14 +102,16 @@ elif [ -d %{_localstatedir}/lib/rpm-state/initramfs/pending ]; then \
         fi \
     done; \
 fi \
+%grub2_post
 %removal_action
 
 %posttrans
 echo "initramfs" %{version}-%{release} "posttrans" >&2
 %removal_action
 mkinitrd -q
-# Move initrd generated for kernel-mshv to /boot/efi, where linuxloader expects to find it
-mv /boot/initrd.img-*mshv* /boot/efi/ >/dev/null 2>&1 || :
+# Copy initrd generated for kernel-mshv to /boot/efi, where linuxloader expects to find it
+cp /boot/initrd.img-*mshv* /boot/efi/ >/dev/null 2>&1 || :
```

Comment:
It's probably a bad idea to ignore failure here, as the ESP could easily be full...

Reply:
Good point, I'll make that fix.